src/assets/blocks/10x10x10palette/data.json

```diff
@@ -270,9317 +275,9317 @@
+      "smiles": "Cc1ccc(-c2c3ccccc3c(N(c3ccccc3)c3ccc4ccccc4c3)c3ccccc23)cc1",
       "lambdaMaxShift": 410,
-      "molecularWeight": 485.6
+      "molecularWeight": 385.5100000000001
     },
     "6:4:0": {
       "key": "6:4:0",
-      "chemicalFormula": "C23H18NO",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1",
+      "chemicalFormula": "",
+      "smiles": "",
       "lambdaMaxShift": 310,
-      "molecularWeight": 324.4
+      "molecularWeight": 324.4030000000001
     },
     "6:4:1": {
       "key": "6:4:1",
       "chemicalFormula": "C29H22N2O3",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1O=[N+](C1=CC=[C+]C=C1)[O-]",
+      "smiles": "COc1cc(-c2ccc([N+](=O)[O-])cc2)cc(N(c2ccccc2)c2ccc3ccccc3c2)c1",
       "lambdaMaxShift": 415,
-      "molecularWeight": 446.5
+      "molecularWeight": 446.5060000000002
     },
     "6:4:2": {
       "key": "6:4:2",
-      "chemicalFormula": "C31H24N2O",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1C12=CC=[C+]C=C1C=CN2",
+      "chemicalFormula": "",
+      "smiles": "",
       "lambdaMaxShift": 360,
-      "molecularWeight": 440.5
+      "molecularWeight": 440.54600000000016
     },
     "6:4:3": {
       "key": "6:4:3",
       "chemicalFormula": "C30H22N2O",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1N#CC1=CC=[C+]C=C1",
+      "smiles": "COc1cc(-c2ccc(C#N)cc2)cc(N(c2ccccc2)c2ccc3ccccc3c2)c1",
       "lambdaMaxShift": 385,
-      "molecularWeight": 426.5
+      "molecularWeight": 426.5190000000002
     },
     "6:4:4": {
       "key": "6:4:4",
       "chemicalFormula": "C27H21NOS",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1[C+]1=CSC=C1",
+      "smiles": "COc1cc(-c2ccsc2)cc(N(c2ccccc2)c2ccc3ccccc3c2)c1",
       "lambdaMaxShift": 340,
-      "molecularWeight": 407.5
+      "molecularWeight": 407.5380000000001
     },
     "6:4:5": {
       "key": "6:4:5",
       "chemicalFormula": "C28H22N2O",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1[C+]1=NC=CC=C1",
+      "smiles": "COc1cc(-c2ccccn2)cc(N(c2ccccc2)c2ccc3ccccc3c2)c1",
       "lambdaMaxShift": 370,
-      "molecularWeight": 402.5
+      "molecularWeight": 402.4970000000001
     },
     "6:4:6": {
       "key": "6:4:6",
       "chemicalFormula": "C30H25NO2",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1COC1=CC=[C+]C=C1",
+      "smiles": "COc1ccc(-c2cc(OC)cc(N(c3ccccc3)c3ccc4ccccc4c3)c2)cc1",
       "lambdaMaxShift": 310,
-      "molecularWeight": 431.5
+      "molecularWeight": 431.5350000000002
     },
     "6:4:7": {
       "key": "6:4:7",
       "chemicalFormula": "C27H21N3O",
-      "smiles": "C1([N-]C2=CC=CC=C2)=CC=C(C=CC=C3)C3=C1COC1=C[C-]=C[C+]=C1[C+]1=NC=CN=C1",
+      "smiles": "COc1cc(-c2cnccn2)cc(N(c2ccccc2)c2ccc3ccccc3c2)c1",
       "lambdaMaxShift": 385,
-      "molecularWeight": 403.5
+      "molec
```

Comment:
this is probably a dumb question on my part, but do we need the chemicalFormula and smiles fields?